umh.docs.umh.app/content/en/docs/getstarted/installation.md

```
@@ -26,15 +26,22 @@ installing your first instance and connecting to an OPC UA simulator in no time.
 
 - ARM-based systems, such as a Raspberry Pi, are not currently supported.
 
+- Ensure the root partition (`/`) has at least <span id="storage-requirements"></span> of free space.
```

Markdownlint (0.37.0)

Element: span
Inline HTML

(MD033, no-inline-html)